tests/tests.toml

```
@@ -676,6 +676,10 @@ asr = true
 filename = "errors/test_list_count.py"
 asr = true
 
+[[test]]
+filename = "errors/test_list_index.py"
+asr = true
```

```suggestion
run = true
```

See, https://github.com/lcompilers/lpython/blob/a1fff8b7460fe350b3edbf837b7ff79c76192781/tests/tests.toml#L1025-L1040